collections/co_3daily_data.yaml

```diff
@@ -22,7 +22,8 @@ Tags:
   - sentinel hub
   - raster
   - race challenges
-License: Full, open and free access
+License: |
```

Comment:
License should have these three entries:
License:
LicenseType:
LicenseUrl:
Since we dont have the url ; I suggest you leave it as it was  `License: Full, open and free access` 

Also you have specified the SH license, which in this case is not correct

Reply:
Done.